pkgs/by-name/ku/kubectl-ai/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule (finalAttrs: {
```

Comment:
```suggestion
buildGoModule rec {
```

Reply:
What no.

Reply:
`rec` to point to versions is deprecated, you shouldn't use it here.

Reply:
Sorry did not review the previous suggestions